simulator/server/server.go

```diff
@@ -46,8 +46,8 @@ func NewSimulatorServer(cfg *config.Config, dic *di.Container) *SimulatorServer
 
 	v1.PUT("/reset", resetHandler.Reset)
 
-	v1.GET("/export", exportHandler.Export)
-	v1.POST("/import", exportHandler.Import)
+	v1.GET("/export", exportHandler.Save)
+	v1.POST("/import", exportHandler.Load)
```

Comment:
I prefer `Snap` instead of `Save`. 
`Save` can be mistakenly interpreted as "take a snapshot file and save it to somewhere, or save all resources into kube-apiserver".